app/models/step_by_step_page.rb

```diff
@@ -41,6 +41,21 @@ def self.validate_redirect(redirect_url)
     redirect_url =~ regex
   end
 
+  # This function first generates a hash from the object id,
```

Comment:
This method can perhaps also be extracted into a separate class. Would be good to document our discussion about what the input for `Digest::SHA256.hexdigest()` is.

Reply:
I can put our discussion into the commit message.

Reply:
I think it would be better to rewrite the comment to be clearer, "The purpose is to generate a deterministic hash" isn't enough because it doesn't explain why.